src/libwasm.rs

```diff
@@ -104,15 +104,14 @@ fn primitive_value_json(primitive: Primitive) -> Value {
 
 type JsFieldElement = JsValue;
 
-// TODO: remove this in favour of the new EntityQuery
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
-pub struct EntityModel {
+pub struct EntityQuery {
```

Comment:
if we're going to rename it to EntityQuery then we should update the struct to match the original Query struct which has support for clauses
```rust
#[derive(Clone, Debug)]
#[repr(C)]
pub struct Query {
    pub limit: u32,
    pub offset: u32,
    pub clause: COption<Clause>,
}
```

like this one which is defined in the cbindgen types